_tools/generate_release_cycle.py

```diff
@@ -28,21 +28,52 @@ def parse_date(date_str: str) -> dt.date:
 class Versions:
     """For converting JSON to CSV and SVG."""
 
-    def __init__(self) -> None:
+    def __init__(self, limit_to_active=False, special_py27=False) -> None:
```

Comment:
Let's make them keyword-only, to require the name and prevent calling `Versions(True, False)`

(We already do call with keywords 👍)

```suggestion
    def __init__(self, *, limit_to_active=False, special_py27=False) -> None:
```